README.md

```diff
@@ -64,15 +64,72 @@ A wrapper around `UXTX` to manage many transactions.
 import { TxManager, TxManagerOptions } from '@bpanel/bpanel-utils';
 
 // use default labels and constants in TxManagerOptions
-const txManager = TxManager.fromOptions(TxManagerOptions);
+(async () => {
+  const txManager = TxManager.fromOptions(TxManagerOptions);
 
-const account = 'default';
-const txs = await wallet.getHistory(account);
+  const account = 'default';
+  const txs = await wallet.getHistory(account);
 
-// generate list of UXTX.toJSON with initial labels and constants
-const parsed = txManager.parse(txs);
+  // generate list of UXTX.toJSON with initial labels and constants
+  const parsed = txManager.parse(txs);
+})()
 ```
 
+#### BPClient
+A utility client to manage connections to a node within your app.
+It is configured to make connections via the proxy server where your bPanel app
+is served from, but should support custom connections either directly to a remote
+node or even one running on the client. It will support connecting to a bcoin, bcash,
+or handshake node.
+
+Example:
+```js
+import { BPClient } from '@bpanel/bpanel-utils';
+
+(async function() {
+  const client = new BPClient({ id: 'test', chain: 'bcoin' });
```

Comment:
What are the valid chains? `bcoin`, `bcash` and `handshake`?

I think that you should have a constant list that holds the valid chains, similar to how there is a networks list

Reply:
just a nit, `bcoin` and `bcash` aren't chains but implementations (like `hsd` vs. `handshake`). I can add a list of available constants on the class same as `this.types`? So `this.chains`?

Reply:
Any preference on data type? Array? Set?

Reply:
Ended up just using the isValidChain utility. Having that, it didn't seem worth it to hard code it into the client especially since as we add more chains we can just do it one place.